server/routes/profile.js

```diff
@@ -1,25 +1,36 @@
 const express = require('express');
 
-const createProfileRouter = () => {
+const createProfileRouter = ({ offenderService }) => {
   const router = express.Router();
 
+  const getPersonalisation = async user => {
+    const events = await offenderService.getEventsForToday(user);
+
+    const timetableError = events?.error;
+    const { morning, afternoon, evening } = events || {};
+    const signedInUser = user.getFullName();
```

Comment:
this signedInUser should probably move into middleware at some point and added to res.locals.